docs/HowToLogInLocally.md

```diff
@@ -0,0 +1,27 @@
+# Authentication in your Development Env. 
+
+In order to be able to log in to your local environment, you will need to do the following: 
+
+## Prerequisites
+
+- Have the [site](https://github.com/department-of-veterans-affairs/vets-website) building locally, and the drupal content updated. 
+- Have the [API](https://github.com/department-of-veterans-affairs/vets-api) running locally, with [mock-api-data](https://github.com/department-of-veterans-affairs/vets-api-mockdata), either in docker or locally
+
+
+## To log in:
+0) Have both the website and API running. 
+1) Head to [flipper for the API](http://localhost:3000/flipper/features)
+2) Disable the `ssoe` feature flag. 
+3) Go your local website, and select sign in and sign in with `ID.me`
+4) Log in with a appropriate user, ask your team for what can be used. 
+5) Click through the next few pages, it won't send a text, and it should allow you to pass right on through. 
+6) You should be redirected back to the home page, signed in.
+7) To see if it worked, you should be able to open your [redux dev tools](https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd) and see the user object in the global state.
```

Comment:
Each item in this list can just start with `1.`  and GitHub MD will automatically enumerate it